Resources/Prototypes/Entities/Structures/Furniture/Tables/tables.yml

```diff
@@ -403,6 +403,47 @@
     footstepSoundCollection:
       collection: FootstepTile
 
+- type: entity
+  id: TableBrass
+  parent: TableBase
+  name: brass table
+  description: A shiny, corrosion resistant brass table. Steampunk!
+  components:
+  - type: Sprite
+    sprite: Structures/Furniture/Tables/brass.rsi
+  - type: Icon
+    sprite: Structures/Furniture/Tables/brass.rsi
+  - type: Damageable
```

Comment:
Why does this have iconcomponent?

Reply:
all of the other tables do
without the iconcomponent the console throws an error, and trying to construct the table will freeze the game